starfish/test/image/imagestack_test_utils.py

```diff
@@ -38,20 +38,21 @@ def verify_stack_fill(
 
 def verify_physical_coordinates(
         stack: ImageStack,
-        selectors: Mapping[Axes, int],
         expected_x_coordinates: Tuple[float, float],
         expected_y_coordinates: Tuple[float, float],
         expected_z_coordinates: Tuple[float, float]
```

Comment:
I would argue that Z coordinates should be allowed to vary throughout the imagestack, and that we should support testing for that.

Reply:
What kind of non-aligned data are you thinking about supporting here? I agree that generality is important, but for the SpaceTx groups this might not be necessary (at least in the short term). The registration that I've seen applied is: 

1. register `(x, y)` tiles across `z` against a single `z` anchor/reference tile
2. register `(x, y, z)` volumes across `(r, c)` against a single `(r, c)` anchor/reference volume